hazelcast-sql/pom.xml

```diff
@@ -476,10 +476,6 @@
                     <groupId>org.jetbrains.kotlin</groupId>
                     <artifactId>kotlin-stdlib</artifactId>
                 </exclusion>
-                <exclusion>
```

Comment:
This is an exclusion, not dependency - I think you didn't meant to remove exclusion (= re-adding dependency)

Reply:
7929dcf09751dfb2c823c774d90104a29ae5e187